### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by ewelinagr**

### CODE REVIEW

File: data-showcase/grails-app/services/nl/thehyve/datashowcase/ProjectService.groovy [hunk 1]
```diff
@@ -7,20 +7,95 @@
...
 
+    List<ProjectRepresentation> getProjects(Set concepts, SearchQueryRepresentation searchQuery) {
+
+        Criterion searchQueryCriterion = searchQuery ? searchCriteriaBuilder.buildCriteria(searchQuery) : null
+        def stopWatch = new StopWatch('Fetch projects filtered items')
+        stopWatch.start('Retrieve from database')
+        // stateless session does not support collections of associated objects
+        // http://forum.spring.io/forum/spring-projects/batch/37785-collections-cannot-be-fetched-by-a-stateless-session
+        def session = sessionFactory.openSession()
+        Transaction tx = null
```
Comment (gijskant) [inline data-showcase/grails-app/services/nl/thehyve/datashowcase/ProjectService.groovy]: But there is already `Transactional` on top of the class. Why is this needed?

Comment (ewelinagr) [reply data-showcase/grails-app/services/nl/thehyve/datashowcase/ProjectService.groovy]: right, I changed a stateless session to the stateful one and I thought this is needed then. I'll remove it.

File: data-showcase/grails-app/services/nl/thehyve/datashowcase/ItemService.groovy [hunk 2]
```diff
@@ -82,23 +94,29 @@ class ItemService {
...
         }
         stopWatch.stop()
-        log.info "Items fetched.\n${stopWatch.prettyPrint()}"
+        log.info "${result.size()} items fetched.\n${stopWatch.prettyPrint()}"
         result
     }
 
     @Transactional(readOnly = true)
-    List<ItemRepresentation> getItems(Set concepts, Set projects, SearchQueryRepresentation searchQuery) {
+    List<ItemRepresentation> getItems(int firstResult, int maxResults, String order, String propertyName,
```
Comment (gijskant) [inline data-showcase/grails-app/services/nl/thehyve/datashowcase/ItemService.groovy]: I guess this can be refactored to one `pageable` object, but that can be done later.
